Cargo.toml

```
@@ -122,7 +122,7 @@ serde = { version = "^1.0.190", optional = true, features = ["derive"] }
 ntapi = { version = "0.4", optional = true }
 # Support a range of versions in order to avoid duplication of this crate. Make sure to test all
 # versions when bumping to a new release, and only increase the minimum when absolutely necessary.
-windows = { version = ">=0.59, <0.62", optional = true }
+windows = { version = ">=0.62, <0.63", optional = true }
```

```suggestion
windows = { version = "0.62", optional = true }
```
for now, it's exactly the same as specify range. if it comes with any incompatibilities in future, we could pin ranges again.

---

true.
The reason, i left the range, was to leave a hint for the next editor.